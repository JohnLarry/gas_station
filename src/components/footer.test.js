import React from "react";
import {shallow} from "enzyme";
import renderer from "react-test-renderer";
import   Footer    from "./footer";

describe("test footer component",()=>{
	it("check the text in h5 tag", ()=>{
		const wrapper = shallow(<Footer/>);
		const text = wrapper.find('h5').text();
		const text1 = wrapper.find('h4').text();
		expect(text).toEqual("Haykpo");
		expect(text1).toEqual("Products");

	});

	it("check if snapshot match",()=>{

		const tree = renderer.create(<Footer/>).toJSON();
		expect(tree).toMatchSnapshot();
	});


});