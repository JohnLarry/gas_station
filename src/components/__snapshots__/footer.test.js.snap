// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test footer component check if snapshot match 1`] = `
<div
  className="row footerTop"
>
  <div
    className="col smallMargin "
  >
    <div>
      <h5>
        Haykpo
      </h5>
      <p>
        Here for you. Here to
      </p>
      <p>
         stay!
      </p>
    </div>
    <div
      className="social-div"
    >
      <img
        alt="facebook logo"
        src="images/facebook.png"
      />
      <img
        alt="twitter logo"
        src="images/twitter.png"
      />
      <img
        alt="instagram logo"
        src="images/instagram.png"
      />
    </div>
  </div>
  <div
    className="col"
  >
    <h4>
      Products
    </h4>
    <p>
      Petrol
    </p>
    <p>
      Diesel
    </p>
    <p>
      Gas
    </p>
  </div>
  <div
    className=" col"
  >
    <h3>
      Company
    </h3>
    <p>
      About
    </p>
    <p>
      Contact
    </p>
    <p>
      Our terms and conditions
    </p>
  </div>
</div>
`;
